Source/RxComponents/LMCCavityModes.cc

```diff
@@ -164,14 +180,14 @@ namespace locust
 		return true;
 	}
 
-    double CavityModes::GetVoltagePhase(unsigned aChannel)
+    std::vector<std::vector<std::vector<std::vector<double>>>> CavityModes::GetVoltagePhase(int aChannel, int l, int m, int n)
```

Comment:
Same comment as when this is called. Personally, I'd vote for using the aChannel, l, m, and n parameters within this definition and just return a scalar. This would match the structure of the SetVoltagePhase() definition below